opm/parser/eclipse/EclipseState/Tables/tests/TableContainerTests.cpp

```diff
@@ -54,10 +55,7 @@ BOOST_AUTO_TEST_CASE( CreateContainer ) {
     BOOST_CHECK_EQUAL( 0 , container.size() );
     BOOST_CHECK_EQUAL( false , container.hasTable( 1 ));
 
-    std::shared_ptr<Opm::SimpleTable> table = std::shared_ptr<Opm::SimpleTable>( new Opm::SimpleTable() );
-    table->initFORUNITTESTONLY(deck->getKeyword("SWOF")->getRecord(0)->getItem(0),
-                              columnNames);
-
+    std::shared_ptr<Opm::SimpleTable> table = std::make_shared<Opm::SwofTable>( deck->getKeyword("SWOF")->getRecord(0)->getItem(0) );
```

Comment:
above you used the type-safe variant of `getKeyword()`, (i.e., `deck->template getKeyword<Opm::SWOF>()`).